### DISCUSSION TITLE
blog/2023/06/01/geocoding-routing-openstreetmap-r/

### DISCUSSION THREAD

**Discussion by giscus** [andrewheiss/ath-quarto#84] [Blog comments]
# blog/2023/06/01/geocoding-routing-openstreetmap-r/

Use R to get geocoded location and routing data from OpenStreetMap and explore our family’s impending 5,000 mile road trip around the USA

https://www.andrewheiss.com/blog/2023/06/01/geocoding-routing-openstreetmap-r/

**Comment by jc2it** [comment]
This is great. As you go through St. Louis pickup some excellent BBQ at Pappy's Smokehouse.

**Reply by jc2it** [reply]
Also, When we take road trips out west we try to have a couple of audio books to pass the miles. Did you think about adding bathroom breaks, fuel stops and places to stretch your legs. Often we travel with children and parks near gas stations work well for this. A 20 minute stop refreshes everyone on a 10 hour road trip.

**Comment by quintablet** [comment]
Excellent article!  I'm going to try it on my next RV trip!

**Comment by scottstanfield** [comment]
Bravo! Turns out you also made a fantastic tutorial on tibble, ggplot2, forcats, dplyr and magrittr. You're using dplyr features like lead, pivot_longer, and virtually every other primary verb. I'm a data.table guy, but I think you've convinced me, at least for legibility (as performance isn't a concern here). I found myself following along in R as I'm reading. 

This reminds me of the old AAA TripTik strip maps.

**Comment by dnstevenson1** [comment]
Very cool. We've been to most of the western sites you mention with our kids. If I might, a couple of suggestions:

1- Get your kids national parks passport books. They will enjoy getting the stamps at each park/monument. https://store.wnpa.org/collectibles-gifts/park-passports.html
2- When you're at Mount Rushmore, hit up Wind Cave National Park nearby. Get a tour of the caves, and even better - see some buffalo up close. Or hit Badlands National Park for excellent wildlife viewing. The kids will love the prairie dogs.
3- Little Bighorn National Battlefield was surprisingly interesting. We visited it the same day as Devil's Tower.

Enjoy, and safe travels!

**Comment by rejexx** [comment]
Not too long ago I moved from Houston, TX to Spanish Fork and made a very similar trip there. Next road trip east, I could up my road trip game with this idea.  Thanks for sharing!
